feg/gateway/services/n7_n40_proxy/n7/model_conversion.go

```
@@ -132,6 +191,15 @@ func getSbiPduSesionType(pduSessionType protos.PduSessionType) sbi.PduSessionTyp
 	}
 }
 
+func getFailureCodeN7(failureCodeProto protos.PolicyReAuthAnswer_FailureCode) *n7_sbi.FailureCode {
+	failureCodeNum := int(failureCodeProto)
+	if failureCodeNum > len(FailureCodeProtoToN7Map) {
+		return nil
```

doing things like that we are hiding errors. 
If for some reason we receive wrong formed value we just ignore, and don't say anything about it. 

I think some functions on `model_conversion` will need to be refactored to trigger error and stop process. It doesnt make any sense to send to AGW a proto that we know it is not formed properly.

So either PCF sends a proper message, or we error

---

Please, let's create another PR to refactor moder_conversion to add error in function that deserve to error. For example this one.

---

> So either PCF sends a proper message, or we error

In this particular case the message is sent by SessionD. Again this comes in a response message from SessionD and we can send back an error to SessionD. We can probably log the error though.